packages/core/src/protobufs/generated/message.ts

```diff
@@ -103,6 +103,10 @@ export enum MessageType {
   FRAME_ACTION = 13,
   /** LINK_COMPACT_STATE - Link Compaction State Message */
   LINK_COMPACT_STATE = 14,
+  /** TAG_ADD - Add a Tag to a Cast */
+  TAG_ADD = 15,
+  /** TAG_REMOVE - Remove a Tag from a Cast */
+  TAG_REMOVE = 16,
```

Comment:
autogen here too?

Reply:
no, this has to be specified unfortunately